src/__tests__/is-of-type.dts.spec.ts

```diff
@@ -0,0 +1,78 @@
+import { isOfType } from '../is-of-type'
+import { createActionCreator } from '../create-action-creator'
+import { ActionType } from '../types'
+
+// @dts-jest:group isOfType
+
+const increment = createActionCreator('[Counter] increment')
+const decrement = createActionCreator('[Counter] decrement')
+const reset = createActionCreator(
+  '[Counter] reset',
+  resolve => (value: number) => resolve(value)
+)
+const actions = [increment, decrement, reset]
```

Comment:
This is the cause of that weird types. action creators should call here to output actions.
```ts
const actions = [increment(), decrement(), reset()]
```

Reply:
This was simply my mistake :scream:
completed!